mk/dev.mk

```diff
@@ -182,6 +183,10 @@ dev/install/goimports: ## Bootstrap: Install goimports
 dev/install/helm3: ## Bootstrap: Install Helm 3
 	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
 
+.PHONY: dev/install/helm-docs
+dev/install/helm-docs: ## Bootstrap: Install Helm Docs
+	go get github.com/norwoodj/helm-docs/cmd/helm-docs
```

Comment:
Is there any way to pin this to a version (even just major)? Maybe we could use [the docker image](https://hub.docker.com/r/jnorwood/helm-docs) as an alternative if not?

Reply:
Actually, there is a binary release available. Let's use that one:
https://github.com/norwoodj/helm-docs/releases

Reply:
sure ! will do it

Reply:
If we taking binaries, make sure to support Linux and MacOS

Reply:
yes, it has support for both